### PR TITLE
AB#822 Create logo with column

### PR DESCRIPTION
**Pull request by szymonbla**

### CODE REVIEW

File: src/common/Sandbox/Sandbox.tsx [hunk 1]
```diff
@@ -2,30 +2,31 @@ import Typography from '@mui/material/Typography';
...
 export function Sandbox() {
   const dispatch = useAppDispatch();
 
   return (
-    <Grid container spacing={2} sx={{ height: '100%' }}>
+    <Grid container spacing={12} sx={{ height: '100%' }}>
+      <Grid item xs={2.5}>
+        <CategoryColumn categories={CategoryListConfig} />
+      </Grid>
       <ConfirmationModal
```
Comment (JKrymarys) [inline src/common/Sandbox/Sandbox.tsx:23]: It's a modal, so let's keep it on very top or very bottom of the JSX

File: src/common/components/CategoryColumn/CategoryList/CategoryItem/CategoryItem.tsx [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import { Grid, Typography } from '@mui/material';
 
-import { ColorfulTagIcon, ColorfulTagIconProps } from 'common/components/CategoryList/ColorfulTagIcon';
+import { ColorfulTagIcon, ColorfulTagIconProps } from 'common/components/CategoryColumn/CategoryList/ColorfulTagIcon';
```
Comment (JKrymarys) [inline src/common/components/CategoryColumn/CategoryList/CategoryItem/CategoryItem.tsx:3]: Why so deep? What's the reason why `ColorfulTagIcon` can't be in `common/components`?

**Review by JKrymarys — APPROVED** [review]
Approved, but two minor suggestions

File: src/common/components/CategoryColumn/CategoryList/CategoryItem/CategoryItem.tsx (hunk 2 above)
Comment (szymonbla) [reply src/common/components/CategoryColumn/CategoryList/CategoryItem/CategoryItem.tsx:3]: I believe that's component strictly connected with `CategoryColumn`, so it should be under mentioned folder. If you like I can adjust to  `import { ColorfulTagIcon, ColorfulTagIconProps } from 'common/components/CategoryColumn/`

**Review by mkidawa — APPROVED** [review]

**Pull request merged**